modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,564 @@
+=== Confirm Continuous Data Balancing configuration
+
+Run xref:reference:rpk/rpk-cluster/rpk-cluster-config-get.adoc[`rpk cluster config get partition_autobalancing_mode`] to ensure that xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing] is configured and enabled.
+
+Example: 
+```bash
+rpk cluster config get partition_autobalancing_mode
+```
+
+Output:
+```bash
+continuous
+```
+
+[[debug-bundle]]
+=== Generate debug bundle
+
+Ensure that you’re able to generate a debug bundle from each host and upload to Redpanda Support.
+
+```bash
+sudo rpk debug bundle
+Creating bundle file...
+Debug bundle saved to '1711372017-bundle.zip'
+```
+
+See also:
+- xref:reference:rpk/rpk-debug/rpk-debug-bundle[]
+- xref:manage:kubernetes/troubleshooting/k-diagnostics-bundle/[]
+
+[[topic-rf]]
+=== Topic replication factor
+
+Ensure that all topics have a replication factor greater than one.
+
+Example:
+```bash
+rpk topic list
+```
+
+Output:
+```bash
+NAME   PARTITIONS  REPLICAS
+bad    1           1
+good   1           3
+```
+
+See also:
+- xref:upgrade:migrate/data-migration.adoc#change-topic-replication-factor[]
+
+[[maintenance-mode]]
+=== Brokers in maintenance mode
+
+Ensure that no brokers are in maintenance mode.
+
+Example:
+
+```bash
+rpk cluster maintenance status
+```
+
+Output:
+
+```bash
+NODE-ID  ENABLED  FINISHED  ERRORS  PARTITIONS  ELIGIBLE  TRANSFERRING  FAILED
+1        false    -         -       -           -         -             -
+2        false    -         -       -           -         -             -
+3        false    -         -       -           -         -             -
+```
+
+See also:
+- xref:manage:node-management.adoc#place-a-broker-in-maintenance-mode[]
+
+[[decom-state]]
+== Brokers in decommissioned state
+
+Ensure that no brokers are in a decommissioned state.
+
+Example:
+```bash 
+rpk redpanda admin brokers list
+```
+
+Output:
+```bash
+NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
+0        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+1        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+2        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+```
+
+See also:
+- xref:manage/cluster-maintenance/decommission-brokers/[]
+
+== Level 2 production readiness
+
+The Level 2 readiness checklist helps you to confirm that:
+
+- You have adhered to 2-day operations best practices
+- You have sufficient ability to diagnose and recover from any issues that arise or failures that occur
+
+Level 2 production readiness ensures that you can monitor and support your environment on a sustained basis, and includes the following checks:
+
+=== Environment configuration
+
+Confirm that you have a development or test environment configured so that you can evaluate upgrades and new versions before rolling them straight to production.
```

Comment:
```suggestion
Confirm that you have a xref:deploy:deployment-option/self-hosted/manual/production/dev-deployment.adoc[development] or test environment configured so that you can evaluate upgrades and new versions before rolling them straight to production.
```